disnake/enums.py

```diff
@@ -503,7 +503,7 @@ class UserFlags(Enum):
     bug_hunter_level_2 = 1 << 14
     verified_bot = 1 << 16
     verified_bot_developer = 1 << 17
-    discord_certified_moderator = 1 << 18
+    moderator_programs_alumni = 1 << 18
```

Comment:
this is breaking, but it can be added as an alias.
```suggestion
    discord_certified_moderator = 1 << 18
    moderator_programs_alumni = 1 << 18
```

Reply:
Additionally the enum docs need to be updated.

Reply:
I will add it now